pkg/eks/fargate_test.go

```diff
@@ -73,4 +73,11 @@ var _ = Describe("Fargate", func() {
 			})
 		})
 	})
+
+	Describe("DoListFargateProfiles", func() {
+		It("should list profiles", func() {
+			fakeClient.ListProfilesReturns([]string{fargateProfileName}, nil)
+			Expect(eks.DoListFargateProfiles(context.Background(), config, fakeClient)).To(Succeed())
+		})
+	})
```

Comment:
this is not required either, can we remove this?

Reply:
rebased `release-0.125.0` and removed unnecessary code here.